src/app.js

```diff
@@ -1 +1,32 @@
 'use strict';
+
+const fs = require('fs');
+const args = process.argv.slice(2);
+
+if (args.length === 2) {
```

Comment:
here you have ONLY two arguments which you sliced from prosecc.argv, so I don't understand why you would need this checking. It's better to check `process.argv.length`

Reply:
If user enter just one? we write an app? and user can be not very smart